### DISCUSSION TITLE
Sudden pricing and resources change, scam?

### DISCUSSION THREAD

**Discussion by fpichardom** [sqlitecloud/discussions#46] [Help]
I have been following this project for several months since late June, and then I was super excited with the announcement of the integration with SQLalchemy as a Python developer. Since then, I have been requesting access to the pro tier, which was, at the time, US $9 and offered more generous resources. I mainly asked for a way to support the project but was never given access. I got a reply, then got wholly ignored after saying that I was running hobby projects and wanted to support the project. But now I see that maybe I was never given access because you didn’t want people in that plan to begin with, as you now increased the price (US $19) and cut resources. Why put it out there if you were not accepting subscriptions? And why advertise something you couldn’t support? I went from super excited to very wary and suspicious of this project. Now, I do not trust any app on it. Such a shame, so much promise and potential. People discovering this project now beware.

**Comment by marcobambini** [comment]
Hi @fpichardom,

I sincerely apologize for the poor communication on our part.

We have not yet announced an official pricing plan, as we've been testing several models to find the best fit for our platform’s architecture. The Pro tier is still in development; we are currently working on the necessary billing, infrastructure, and services.

Our goal is to launch the product for general availability in November, but until then, the Pro plan will remain unavailable, and the pricing page will continue to evolve.

I want to emphasize that everything you see at this point is a work in progress, with many changes and adjustments expected. This is why there’s a "Currently in Beta" banner on the homepage. We’ll remove that banner and introduce a finalized Pro plan and pricing structure when we officially launch.

Again, I apologize for any confusion this may have caused, and I sincerely hope we can restore your trust. Please feel free to reach out if you have any further questions or concerns.

**Comment by fpichardom** [comment]
Hi @marcobambini,

Thanks for the quick response. I want to say that maybe I entered the conversation too harshly, and that was a bit unfair. It is just that the price changes shocked me when I entered the site yesterday. I have seen so many cases of rug pull in tech recently, and I’m starting to get tired of it. I’m starting to have a more wary stance on new companies promising too much then suddenly changing the rules, and this was starting to seem like one. I prefer a company that carefully thinks through what they can support and communicates clearly. I know the site says it is in beta, but it could have been that you already have a paying plan, as I feel the site is misleading. For example, at the top of the pro version, it says the most popular, as if people are already using it. I would signal on the page that this plan is not currently available, like a banner saying coming soon, in development, etc. And a notice indicating that pricing and services are subject to change during the beta. It doesn't need to be the way I mentioned, but some sort of clarification would be appreciated, be creative. I would still be willing to use the product as long as the company is respectful to their customers and provides a great product. I like the concept of this project and I genuinely would like to see it succeed.

**Comment by marcobambini** [comment]
@fpichardom we updated the [Pricing](https://sqlitecloud.io/pricing) page to reflect our current status better.
Thanks a lot for your invaluable feedback.
